sdk/storage/Azure.Storage.Files.Shares/src/Models/ShareFileRenameOptions.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Metadata = System.Collections.Generic.IDictionary<string, string>;
+
+namespace Azure.Storage.Files.Shares.Models
+{
+    /// <summary>
+    /// Optional parameters for renaming a File or Directory.
+    /// </summary>
+    public class ShareFileRenameOptions
```

Comment:
If these options are going to be used for both ShareFile and ShareDirectory, should the name be more like SharePathRenameOptions? I don't have a strong opinion on the name.

Reply:
Let's see what the arch board says.